### PR TITLE
science/lenstool: submission new Portfile

### PR DESCRIPTION
**Pull request by ejullo**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix
- [x] submission (new Portfile) 

###### Tested on

macOS 10.13.6 17G5019
Xcode 9.1 9B55 

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?

- [x] checked your Portfile with `port lint`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by cjones051073** [comment]
Thanks, but you did not correctly follow the commit guidelines

https://trac.macports.org/wiki/CommitMessages

commit comments should have the form "port-name : comment"

File: science/lenstool/Portfile [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+PortSystem          1.0
+PortGroup           compilers 1.0
+
+name                lenstool
+version             7.1.1
+categories          science
+license             GPL
+platforms           darwin
+maintainers         lam.fr:eric.jullo openmaintainer
```
Comment (cjones051073) [inline science/lenstool/Portfile]: please add your github id here. i.e.

maintainers {lam.fr:eric.jullo @ejullo} openmaintainer

**Comment by cjones051073** [comment]
b.t.w. I see you have used the master branch of your fork for this PR. Please not this is generally a bad idea, as it means you cannot update your master (with other changes) before this PR is merged.

Standard practise is to first create a new feature branch in your fork, push the cahnges to that then use that feature branch for this PR. This is better as then you can have as many feature branches as you want, for independent PRs etc.

File: science/lenstool/Portfile (hunk 1 above)
Comment (ejullo) [reply science/lenstool/Portfile]: Thanks for your quick reply. I've done the change in the Portfile

Comment (cjones051073) [reply science/lenstool/Portfile]: Thanks, but you also need to squash your commits, so there is only one in this PR, that follows correctly follows the commit guidelines.

**Comment by cjones051073** [comment]
why have you just closed this PR ?

**Comment by ejullo** [comment]
I don't really know what happened. I am going to create a new one with just 1 commit. Sorry, I'm not so fluent in Git to squash commits. I think I messed up with the previous ones.

**Pull request closed without merging**